src/app/tests/TestCommandInteraction.cpp

```diff
@@ -161,7 +161,7 @@ void DispatchSingleClusterCommand(const ConcreteCommandPath & aCommandPath, chip
         }
         else
         {
-            apCommandObj->PrepareCommand(aCommandPath);
+            apCommandObj->PrepareCommand(aCommandPath, aCommandPath);
```

Comment:
Suggest using two helper variables (const references), one for original request path and one for response path, so that the same arg does not appear twice.